draft-ietf-oauth-status-list.md

```diff
@@ -86,69 +94,137 @@ Relying Party:
+This section specifies Status List Tokens in Json Web Token (JWT) and Cbor Web Token (CWT) format.
 
-3. The claim value object MUST contain a "lst" (list) member with a string value that represents the status values for all the tokens it conveys statuses for. The value MUST be computed using the algorithm described in [](#jwt-status-list-claim-encoding).
+## Status List Token in JWT Format {#status-list-token-jwt}
 
-### Status List Encoding {#jwt-status-list-claim-encoding}
+The Status List Token MUST be encoded as a "JSON Web Token (JWT)" according to {{RFC7519}}.
 
-Each status of a Referenced Token MUST be represented with a bit size of 1,2,4, or 8. Therefore up to 2,4,16, or 256 statuses for a Referenced Token are possible, depending on the bit size. This limitation is intended to limit bit manipulation necessary to a single byte for every operation and thus keeping implementations simpler and less error prone.
+The following content applies to the JWT Header:
 
-1. The overall Status List is encoded as a byte array. Depending on the bitsize, each byte corresponds to 8/(#bit-size) statuses (8,4,2, or 1). The status of each Referenced Token is identified using the index that maps to one or more specific bits within the byte array. The index starts counting at 0 and ends with "size" - 1 (being the last valid entry). The bits within an array are counted from least significant bit "0" to the most significant bit ("7"). All bits of the byte array at a particular index are set to a status value.
+* `typ`: REQUIRED. The JWT type MUST be `statuslist+jwt`.
 
-2. The complete byte array is compressed using the "DEFLATE" {{RFC1951}} compression method and stored using the "ZLIB" {{RFC1950}} data format. Implementations are RECOMMENDED to use the highest compression level available.
+The following content applies to the JWT Claims Set:
 
-3. The result of the compression is then base64url-encoded, as defined in Section 2 of {{RFC7515}}.
+* `iss`: REQUIRED. The `iss` (issuer) claim MUST specify a unique string identifier for the entity that issued the Status List Token. In the absence of an application profile specifying otherwise, compliant applications MUST compare issuer values using the Simple String Comparison method defined in Section 6.2.1 of {{RFC3986}}. The value MUST be equal to that of the `iss` claim contained within the Referenced Token.
+* `sub`: REQUIRED. The `sub` (subject) claim MUST specify a unique string identifier for that Status List Token. The value MUST be equal to that of the `uri` claim contained in the `status` claim of the Referenced Token.
+* `iat`: REQUIRED. The `iat` (issued at) claim MUST specify the time at which the Status List Token was issued.
+* `exp`: OPTIONAL. The `exp` (expiration time) claim MAY convey the time at which it is considered expired by its issuer.
+* `status_list`: REQUIRED. The `status_list` (status list) claim MUST specify the Status List conforming to the rules outlined in [](#status-list-json).
```

Comment:
Im unsure whether this layout of a bullet list of claims is any clearer then the number bullet points we had previously, any particular reason for this change?

Reply:
it allows to describe nested claims and shows more clearly the structure.
Therefore its more dense making the document shorter and better to read imo

Reply:
also it makes things more obvious when REQUIRED/OPTIONAL is the first word in the description